src/components/labelPrinting/LabelPrintingForm.vue

```diff
@@ -51,11 +56,23 @@
           <DataFetcher :fetcher="fetchPrinters">
             <fieldset>
               <traction-heading level="3" show-border>Choice of Printer</traction-heading>
+              <traction-muted-text>Label type</traction-muted-text>
```

Comment:
Just a thought- As there are multiple dropdowns to select in a section, I think that users can easily identify what each field if we use proper labels above each dropdown for e.g 'Select Label type" and  " Select Printer" instead of having muted texts ("Label type" and "The printer to print the labels") ? What do you think?